packages/discord.js/src/structures/Attachment.js

```diff
@@ -99,6 +99,26 @@ class Attachment {
      * @type {boolean}
      */
     this.ephemeral = data.ephemeral ?? false;
+
+    if ('duration_secs' in data) {
+      /**
+       * The duration of this attachment in seconds (if an audio)
```

Comment:
```suggestion
       * The duration of this attachment in seconds
       * <info>This will only be available if the attachment is an audio file</info>
```